compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -159,6 +159,9 @@ pub fn to_llvm_features<'a>(sess: &Session, s: &'a str) -> SmallVec<[&'a str; 2]
         ("x86", "rdrand") => smallvec!["rdrnd"],
         ("x86", "bmi1") => smallvec!["bmi"],
         ("x86", "cmpxchg16b") => smallvec!["cx16"],
+        // FIXME: These aliases are misleading, and should be removed before avx512_target_feature is
+        // stabilized. For now, they must be kept at a minimum because std::arch uses them.
```

Comment:
Mildly confusing wording. Perhaps
```suggestion
        // stabilized. They must remain until std::arch switches off them.
```